### ISSUE TITLE
Feat: Changes with register and login page

### ISSUE THREAD

**Issue by kanak227** [Research-Nexas/Unify#426]
### Describe the feature

The UI of login and register page needs improvement . I want to improve it by adding a smooth transition over both the pages in the new UI. Also the separate buttons appearing can be merged into one in the navbar section.

Also as an additional bug the contact , signup and login buttons are not working can fix this too.

Assign it to me for SWOC

### Add ScreenShots

![Screenshot 2024-12-29 200208](https://github.com/user-attachments/assets/a6e304a2-86c8-473f-a7af-1250d15535e5)
![Screenshot 2024-12-29 200252](https://github.com/user-attachments/assets/5dc490a6-38b7-4396-b965-74a3638ab04a)
![Screenshot 2024-12-29 200303](https://github.com/user-attachments/assets/96d92b1a-7cfd-432c-ac27-e670721af89b)

Kind of this page with hover effect over the pages

### Record

- [X] I agree to follow this project's Code of Conduct
- [ ] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by Harshdev098** [comment]
> Also as an additional bug the contact , signup and login buttons are not working can fix this too.
> 

@kanak227  what bug are you talking about. Does the signup and login btns are not working in the login page?

**Comment by kanak227** [comment]
Yes the signup and login buttons are not working because of the wrong linking of html files , also including this the UI of the pages should be made dynamic as attached ok ss

**Comment by kanak227** [comment]
Can you assign me this issue

**Comment by Harshdev098** [comment]
Already assigned you!

**Comment by kanak227** [comment]
I mean the label

**Comment by Harshdev098** [comment]
Will add it till night

**Comment by kanak227** [comment]
Okay

**Comment by Harshdev098** [comment]
Will add other labels after releasing of guidelines of SWoC

**Comment by kanak227** [comment]
Sure

**Comment by prajwal2431** [comment]
its Working login button in login page is leading to form filling page there is not the issue for something

**Comment by kanak227** [comment]
The issue is for UI enhancement. It is good if the login button issue is solved
